cargo-concordium/src/context.rs

```diff
@@ -0,0 +1,122 @@
+use anyhow::anyhow;
+use concordium_contracts_common::{
+    AccountAddress, Address, Amount, ContractAddress, OwnedPolicy, SlotTime,
+};
+use wasm_chain_integration::{ExecResult, HasChainMetadata, HasInitContext, HasReceiveContext};
+
+/// A chain metadata with an optional field.
+/// Used when simulating contracts to allow the user to only specify the
+/// necessary context fields.
+#[derive(serde::Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct ChainMetadataOpt {
+    slot_time: Option<SlotTime>,
+}
+
+impl HasChainMetadata for ChainMetadataOpt {
+    fn slot_time(&self) -> ExecResult<SlotTime> { unwrap_ctx_field(self.slot_time, "slot_time") }
```

Comment:
Shouldn't that be `slotTime` in order that the error message will be meaningful in the context of a JSON context?